code/modules/antagonists/eldritch_cult/knowledge.dm/ash_lore.dm

```diff
@@ -0,0 +1,202 @@
+/datum/eldritch_knowledge/base_ash
+	name = "Nightwatcher's secret"
+	desc = "Opens up the path of ash to you. Allows you to transmute a match with a knife into an ashen blade."
+	gain_text = "City guard knows their watch. If you ask them at night they may tell you about the ashy lantern."
+	banned_knowledge = list(/datum/eldritch_knowledge/base_rust,/datum/eldritch_knowledge/base_flesh,/datum/eldritch_knowledge/rust_final,/datum/eldritch_knowledge/flesh_final)
```

Comment:
question: what happens if you've already researched the final flesh or rust spells when you finally decide to take the basic ash spell?

Reply:
you can only choose 1 basic spell, either ash,rust or flesh. This also locks you into a type of victory, so if you choose rust, your final ritual must be rust.